Source/stores.cpp

```diff
@@ -2402,10 +1994,7 @@ void StartStore(talk_id s)
 		break;
 	case STORE_WBUY:
 		if (storenumh > 0)

```

Comment:
Do we need this check at all?
Can the witch have no items to sale?

If no: remove the check
if yes: do the same as blacksmith.

Reply:
no, she has infinite mana